src/tests/ReactElementAdapter.spec.jsx

```diff
@@ -278,7 +278,8 @@ describe('ReactElementAdapter', () => {
         it('returns the expect.it function for an expect.it content assertion', () => {
 
             const component = <span>{expect.it('to be a string')}</span>;
-            expect(adapter.getChildren(component), 'to equal', [ expect.it('to be a function')] )
```

Comment:
Ah :) OK, I will rewrite it and repush. Currently on the react-render-hook repo :)